wallet/wallet.go

```diff
@@ -33,13 +33,13 @@ func (w *Wallet) Bytes() (secretKey []byte, pubkey []byte) {
 	return secretBytes, pubkeyBytes
 }
 
-func SecretToWallet(secretKey []byte, pubkey []byte) (wallet Wallet, err error) {
+func SecretToWallet(secretKey []byte, pubkey []byte, domain [32]byte) (wallet Wallet, err error) {
```

Comment:
(Not in this PR)
pubkey is not required here if we use NewKeyPairFromSecret in line 38.